src/data/D2ApiIndicator.ts

```diff
@@ -238,4 +85,223 @@ export class D2ApiIndicator {
     private getValueOrEmpty(value: Maybe<string>): string {
         return value ?? "";
     }
+
+    private getIndicatorGroupsByCompetencies(competencies: string[]) {
+        return apiToFuture(
+            this.api.models.indicatorGroups.get({
+                fields: {
+                    id: true,
+                    code: true,
+                    displayName: true,
+                    indicators: {
+                        code: true,
+                        attributeValues: { attribute: { id: true }, value: true },
+                        denominator: true,
+                        displayName: true,
+                        id: true,
+                        indicatorGroups: {
+                            displayName: true,
+                            id: true,
+                            indicatorGroupSet: { id: true, displayName: true },
+                        },
+                        numerator: true,
+                    },
+                },
+                filter: { name: { in: competencies } },
+            })
+        );
+    }
+
+    private buildOutcomeIndicators(
+        competencies: CoreCompetency[],
+        indicatorGroup: D2IndicatorGroup,
+        config: D2Config
+    ): Indicator[] {
+        const competency = competencies.find(
+            c => c.name.toLowerCase() === indicatorGroup.displayName.toLowerCase()
```

Comment:
What's the idea of comparing a name with displayName? will this work as expected when indicatorGroup name has translations?